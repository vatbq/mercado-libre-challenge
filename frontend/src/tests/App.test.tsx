import { render } from "@testing-library/react";
import App from "../App";

it("renders ProgramCard unchanged", () => {
  const { container } = render(<App />);

  expect(container).toMatchSnapshot();
});
