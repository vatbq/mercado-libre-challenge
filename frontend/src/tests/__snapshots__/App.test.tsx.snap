// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ProgramCard unchanged 1`] = `
<div>
  <div
    class="root"
  >
    <img
      alt="Mercado Libre Logo"
      class="logo"
      src="MLLogo.png"
    />
    <div
      class="root"
    >
      <input
        class="inputSearch"
        placeholder="Nunca dejes de buscar"
        type="text"
        value=""
      />
      <button
        class="buttonSearch"
      >
        <img
          alt="Icono de busqueda"
          class="searchIcon"
          src="searchIconBig.png"
        />
      </button>
    </div>
  </div>
</div>
`;
